x-pack/metricbeat/module/oracle/_meta/kibana/7/dashboard/Metricbet-Oracle-tablespaces.json

```
@@ -0,0 +1,536 @@
+{
+  "objects": [
+    {
+      "attributes": {
+        "description": "",
+        "hits": 0,
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "query": {
+              "language": "kuery",
+              "query": ""
+            }
+          }
+        },
+        "optionsJSON": {
+          "hidePanelTitles": false,
+          "useMargins": true
+        },
+        "panelsJSON": [
+          {
+            "embeddableConfig": {},
+            "gridData": {
+              "h": 15,
+              "i": "1",
+              "w": 19,
+              "x": 0,
+              "y": 0
+            },
+            "panelIndex": "1",
+            "panelRefName": "panel_0",
+            "title": "Avg data file size by filename",
+            "version": "8.0.0-SNAPSHOT"
+          },
+          {
+            "embeddableConfig": {},
+            "gridData": {
+              "h": 15,
+              "i": "2",
+              "w": 24,
+              "x": 0,
+              "y": 15
+            },
+            "panelIndex": "2",
+            "panelRefName": "panel_1",
+            "title": "Tablespace Total Size",
+            "version": "8.0.0-SNAPSHOT"
+          },
+          {
+            "embeddableConfig": {},
+            "gridData": {
+              "h": 15,
+              "i": "3",
+              "w": 19,
+              "x": 19,
+              "y": 0
+            },
+            "panelIndex": "3",
+            "panelRefName": "panel_2",
+            "title": "Maximum data file size",
+            "version": "8.0.0-SNAPSHOT"
+          },
+          {
+            "embeddableConfig": {},
+            "gridData": {
+              "h": 15,
+              "i": "4",
+              "w": 10,
+              "x": 38,
+              "y": 0
+            },
+            "panelIndex": "4",
+            "panelRefName": "panel_3",
+            "title": "Ratio of used space in Tablespaces",
+            "version": "8.0.0-SNAPSHOT"
+          },
+          {
+            "embeddableConfig": {},
+            "gridData": {
+              "h": 15,
+              "i": "5",
+              "w": 24,
+              "x": 24,
+              "y": 15
+            },
+            "panelIndex": "5",
+            "panelRefName": "panel_4",
+            "title": "Ratio of used space in data files",
+            "version": "8.0.0-SNAPSHOT"
+          }
+        ],
+        "timeRestore": false,
+        "title": "[Metricbeat Oracle] Overview",
```

I miss some things that I would expect on a overview dashboard, I would name it as `Tablespaces`. And when we have more things we can decide if adding more visualizations to this one and rename it to `Overview`, or create a new overview one and reuse some of the visualizations that we have here.

```suggestion
        "title": "[Metricbeat Oracle] Tablespaces",
```

---

Changed